src/ts/src/common/components/library/data_table/index.module.scss

```diff
@@ -3,20 +3,38 @@
 @import "src/common/styles/spacing";
 @import "src/common/styles/corners";
 
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
 .header {
+  display: flex;
+  // (thuang): Account for Chrome scrollbar width
```

Comment:
is this a TODO?

Reply:
Oh! Sorry the wording is confusing lol `padding-right: $space-l;` is for offsetting the Chrome scrollbar width, so not a TODO. I just changed it to:

```ts
// (thuang): Offset for Chrome scrollbar width
padding-right: $space-l;
```

Reply:
Typically when I see a login name in a comment, it's a TODO.  I'm curious what is the convention being used here.

Reply:
Ooh I see! I typically leave my login name in comments, so it's easier for other readers to know who left the comments and therefore easier to track down the person who has the most context about the comment. I don't think it's a hard requirement, but engineers in the past projects do the same thing!

Reply:
For todo, the format I use in the past is `TODO(login name): TICKET-NUMBER short description` 😄